### ISSUE TITLE
`no-navigation-without-resolve` does not allow fragment only

### ISSUE THREAD

**Issue by al3xnag** [sveltejs/eslint-plugin-svelte#1537]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.

### What version of ESLint are you using?

10.4.0

### What version of `eslint-plugin-svelte` are you using?

3.17.1

### What did you do?

```js
import { replaceState } from '$app/navigation';
...
replaceState(`#hash`, {}); // Unexpected replaceState() call without resolve(). (eslint svelte/no-navigation-without-resolve)
```

### What did you expect to happen?

No lint error

### What actually happened?

Unexpected replaceState() call without resolve(). (eslint svelte/no-navigation-without-resolve)

### Link to **GitHub Repo** with Minimal Reproducible Example

Will provide later if needed.

**Comment by marekdedic** [comment]
Thanks, this is a (fuzzy) duplicate of #1327 and will be fixed by #1482
